lib/ain-grpc/src/call_request.rs

```diff
@@ -173,3 +173,20 @@ pub fn override_to_overlay(r#override: BTreeMap<H160, CallStateOverride>) -> Ove
 
     overlay
 }
+
+#[derive(Clone, Debug, Default, Eq, PartialEq, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
```

Comment:
Will we ever need to `Deserialize` `AccessListResult` ? 
It seems like `#[serde(deny_unknown_fields)]` is not needed

Reply:
Yup. The jsonrpsee server requires the data types to have the Deserialize trait, evn though we dont ever deserialize it on our application implementation level.

Reply:
What about `#[serde(deny_unknown_fields)]` though ? We can remove this